utils/errors/errors.go

```diff
@@ -172,6 +178,15 @@ func WrapWithSkip(err error, msg string, skip int) ErrorExt {
 	return WrapWithSkipAndStatus(err, msg, skip+1, nil)
 }
 
+// WrapWithGRPCCode wraps an existing error and appends stack information if it does not exists along with GRPC status with the given GRPC code
```

Comment:
it looks ok to me. I was thinking about putting this more detailed use case in every service and maybe an organization can create an internal package to wrap it up. But as Carousell is an example, we usually directly use Orion's error package. I think in this small case, it's nice to let Orion has the convenience. @edwardsamuel any concern?